data/saas/dataset/domo_dataset.yml

```diff
@@ -0,0 +1,41 @@
+dataset:
+  - fides_key: domo_connector_example
+    name: Domo Dataset
+    description: A sample dataset representing the Domo connector for Fidesops
+    collections:
+      - name: users
+        fields:
+          - name: id
+            data_categories: [user.derived.identifiable.unique_id]
+            fidesops_meta:
+              primary_key: True
+          - name: alternateEmail
+            default_value: "maskedAlt*@company.com"
```

Comment:
You can omit the `default_value` when describing the fields in a dataset collection. We used this in the legacy spec to define masking rules but that information doesn't need to be specified in the Fidesops dataset.

Reply:
Done